src/web/clients/api/rate-limited.md

```diff
@@ -30,7 +30,7 @@ header! { (XRateLimitLimit, "X-RateLimit-Limit") => [usize] }
 header! { (XRateLimitRemaining, "X-RateLimit-Remaining") => [usize] }
 header! { (XRateLimitReset, "X-RateLimit-Reset") => [u64] }
 
-fn run() -> Result<()> {
+fn main() -> Result<()> {

```

Comment:
This example requires slightly larger rewrite. It was never ideal with recursive `run` but rename to `main` has made it even more apparent. Some minimal `main` should be extracted that calls possibly still recursive `run` (I'd most likely appreciate rewriting it to iterative form).